ci/builders/mac_unopt.json

```diff
@@ -24,7 +24,8 @@
                 "--unoptimized",
                 "--no-lto",
                 "--prebuilt-dart-sdk",
-                "--enable-impeller-3d"
+                "--enable-impeller-3d",
+                "--no-goma"
```

Comment:
Are the two targets that should be experimented are `ios_engine` and `host_engine`?

Reply:
I'm suspecting that any build on mac could trigger it.